### PR TITLE
Introduction to InfluxDB

### LINKED ISSUE #1114: Introduction to InfluxDB
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.
Time-series databases are very popular for data analytics and visualization. In this tutorial, I will explore the basics of time-series databases and where they can be used. I will also cover the basics of InfluxDB, how to set it up and get it running on your local system.

### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
1. Understand time-series data and time-series databases
2. Understand why and where time-series databases can be used
3. Grasp the basics of InfluxDB
4. Setup InfluxDB and run it on your local system
5. Store some data in the database and execute basic queries

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)
https://www.influxdata.com/

### PR DESCRIPTION
**Pull request by adithbharadwaj**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

This fixes #1114

### CODE REVIEW

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

File: articles/introduction-to-influxdb/index.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+Let us consider a few examples to understand this better. The stock market is a very volatile and time-dependent system. It is essential to monitor and store this information along with the timestamp of when the transactions happened. Another example would be gathering sensor data on temperature, pressure, or humidity. These data types require a time associated with each record to help us understand the "trend" or analyze a pattern that changes over time. However, one might ask: Why can't we just add a column/field called time and keep track of the timestamp when the event occurred. This is possible, but there are two ways to do this: When we get a new record or data for a particular value such as temperature, should we overwrite the previous reading, or should we create a new row in the database? Both methods will provide the current or most recent values. However, we can only analyze the trend or the system changes over a period of time if we insert a new row. This is what time-series databases are optimized to do.
```
Comment (lalith1403) [inline articles/introduction-to-influxdb/index.md:19]: However, we can only analyze the trend or the system changes? is it the trend of the system changes? If not, the last sentence needs minor restructuring.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
A minor change is required. Once done, I will check the code.

**Comment by adithbharadwaj** [comment]
> A minor change is required. Once done, I will check the code.

Thanks or the changes @lalith1403 . Yeah that was supposed to be of

**Review by lalith1403 — APPROVED** [review]
The installation and the database commands work fine.

**Comment by hectorkambow** [comment]
plagiarism check done

**Pull request merged**